spec/components/topic_query_spec.rb

```diff
@@ -861,6 +861,59 @@ class ::TopicQuery
     end
   end
 
+  context 'list_unseen' do
+    it "returns an empty list when there aren't topics" do
+      expect(topic_query.list_unseen.topics).to be_blank
+    end
+
+    it "doesn't return topics that were bumped last time before user joined the forum" do
+      user.first_seen_at = 10.minutes.ago
+      create_topic_with_three_posts bumped_at: 15.minutes.ago
```

Comment:
```suggestion
      create_topic_with_three_posts(bumped_at: 15.minutes.ago)
```

Minor code style nitpick. In our codebase, we tend not to leave out brackets in general.